### PR TITLE
[REVIEW] New README

### PR DESCRIPTION
**Pull request by BradReesWork**
add missing text on how to build, section on available algorithms, and sample block

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,27 +1,92 @@
 # <div align="left"><img src="img/rapids_logo.png" width="90px"/>&nbsp;cuGraph - GPU Graph Analytics</div>
 
-[![Build Status](http://18.191.94.64/buildStatus/icon?job=cugraph-master)](http://18.191.94.64/job/cugraph-master/)  [![Documentation Status](https://readthedocs.org/projects/cugraph/badge/?version=latest)](https://docs.rapids.ai/api/cugraph/nightly/)
```
Comment (afender) [inline README.md]: I actually find this useful (when it is working...). Also, cuDF and cuML have it. Is there something preventing cuGraph from using this?

File: README.md [hunk 2]
```diff
@@ -1,27 +1,92 @@
...
+## Getting cuGraph
 ### Intro
 There are 4 ways to get cuGraph :
 1. [Quick start with Docker Demo Repo](#quick)
 1. [Conda Installation](#conda)
 1. [Build from Source](#source)
 
+
+
+Building from source is currently the only viable option. Once version 0.6 is release, the other options will be available.
```
Comment (afender) [inline README.md]: This should be removed

File: README.md [hunk 3]
```diff
@@ -40,20 +105,276 @@ conda install -c nvidia -c rapidsai -c numba -c conda-forge -c defaults cugraph
 
 Note: This conda installation only applies to Linux and Python versions 3.6/3.7.
 
+
+
 <a name="source"></a>
 
+### Build from Source 
+
```
Comment (afender) [inline README.md]: I thought this was supposed to be in `CONTRIBUTING.md` now? cuDF and cuML already adopted the `CONTRIBUTING.md` solution for build and test instructions.

File: README.md (hunk 1 above)
Comment (BradReesWork) [reply README.md]: it has not been working so I dropped it for now.  We can debug the issue and get it working on v0.8

File: README.md (hunk 3 above)
Comment (BradReesWork) [reply README.md]: Currently there is no link to Contributing, and that text does not explain how to build.
This is another punt to v0.8 :(

Comment (BradReesWork) [reply README.md]: oops, was looking at 0.8 branch

**Pull request merged**